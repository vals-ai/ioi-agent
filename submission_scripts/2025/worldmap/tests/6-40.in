version https://git-lfs.github.com/spec/v1
oid sha256:39577467f2862aad276cb7feb4cb4d86c6ae05a386a411c65385bc5d1239d04e
size 1116
