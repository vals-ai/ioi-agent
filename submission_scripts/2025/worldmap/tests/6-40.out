version https://git-lfs.github.com/spec/v1
oid sha256:24bafe147afd7c7cc812064f489878ced80b44f8a9d19e167dd872c72c87b1b9
size 22676
